### PR TITLE
Cockpit 199

### PR DESCRIPTION
**Pull request by KKoukiou**
Rendered preview https://kkoukiou.github.io/cockpit-project.github.io/blog/cockpit-199.html

### CODE REVIEW

File: _posts/2019-07-24-cockpit-199.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 199.
+
+### New logs design
+
+Restyled logs across all of Cockpit.
+
+![service-logs](/images/service-logs.png)
+
+### Design and accesibility improvements on Service page
```
Comment (martinpitt) [inline _posts/2019-07-24-cockpit-199.md]: Let's please continue the tradition of "Page: Description", i. e "Services: Design and accesibility improvements". This is a bit different for the logs design as it affects more than one page.

File: _posts/2019-07-24-cockpit-199.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+The Service Details page now has an indicator that a service status is changing.
```
Comment (martinpitt) [inline _posts/2019-07-24-cockpit-199.md]: Without already knowing the details (and indeed I forgot them, sorry), I don't know what that means. The screenshot does not seem to show anything "changing"?

File: _posts/2019-07-24-cockpit-199.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+In addition the tabs in Service page are now accessible.
```
Comment (martinpitt) [inline _posts/2019-07-24-cockpit-199.md]: There are no tabs on the screenshot, so perhaps this sentence should move below the screenshot?

File: _posts/2019-07-24-cockpit-199.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+![service-details](/images/service-details.png)
+
+### Show DIMM information on Hardware Info page
```
Comment (martinpitt) [inline _posts/2019-07-24-cockpit-199.md]: Likewise, "System:" prefix.

File: _posts/2019-07-24-cockpit-199.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+The Hardware Info page now can show information about DIMM memory, such as size, rank, speed etc.
+
+![dimm-memory](/images/dimm-hwinfo.png)
+
+### VM creation dialog now shows the recommended memory for the selected OS
```
Comment (martinpitt) [inline _posts/2019-07-24-cockpit-199.md]: Machines:

File: _posts/2019-07-24-cockpit-199.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+### New logs design
+
+Restyled logs across all of Cockpit.
+
+![service-logs](/images/service-logs.png)
+
+### Services: Design and accesibility improvements
+
+The enablement and runtime status of units is now represented in a clearer way.
```
Comment (garrett) [inline _posts/2019-07-24-cockpit-199.md]:
```suggestion
Services and other systemd units now sport a simpler on/off state with a clear status overview.
```

File: _posts/2019-07-24-cockpit-199.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+![service-details](/images/service-details.png)
+
+In addition the tabs in Service page are now accessible.
```
Comment (garrett) [inline _posts/2019-07-24-cockpit-199.md]:
```suggestion
In addition, the tabs in Service page are now accessible.
```

File: _posts/2019-07-24-cockpit-199.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+### System: Show DIMM information on Hardware Info page
+
+The Hardware Info page now can show information about DIMM memory, such as size, rank, speed etc.
```
Comment (garrett) [inline _posts/2019-07-24-cockpit-199.md]: "DIMM memory" is redundant, as one of the Ms stands for "memory".

I'd suggest:

```suggestion
The Hardware Info page now displays memory-related information for DIMMs, such as: size, rank, speed, and other details.
```

File: _posts/2019-07-24-cockpit-199.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+![dimm-memory](/images/dimm-hwinfo.png)
+
+_Thanks to Akshit Dua for this feature!_
+
+### Machines: VM creation dialog now shows the recommended memory for the selected OS
+
+The VM creation dialog will now validate the memory field in order to be above
+the allowed minimum and will also show the recommended value for the selected OS.
```
Comment (garrett) [inline _posts/2019-07-24-cockpit-199.md]: Ugh, another broken line comment. We really should not hard wrap in Markdown files. It makes editing them more difficult. Generally, relying on an editor to wrap lines is the way to go for written text, and is what Markdown, AsciiDoc, and other formats recommend.

Anyway:

```suggestion
During VM creation, memory fields are checked for validity. The Create VM dialog now displays a recommended RAM size for the selected OS and an error appears if memory is set too low.
```

File: _posts/2019-07-24-cockpit-199.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+### Try it out
+
+Cockpit 199 is available now:
+
+ * [For your Linux system](https://cockpit-project.org/running.html)
+ * [Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/199)
+ * [Fedora 30](https://bodhi.fedoraproject.org/updates/FEDORA-2019-2284294b43)
+
+*[VM]: Virtual Machine
+*[DIMM]: Dual Inline Memory Module
```
Comment (garrett) [inline _posts/2019-07-24-cockpit-199.md]:
```suggestion
*[DIMM]: Dual Inline Memory Module
*[RAM]: Random Access Memory
*[OS]: Operating System
```

Added RAM and OS.

**Comment by garrett** [comment]
I'm fixing up the screenshots now.

1. Do _not_ use "localhost" as the hostname
2. Try to minimize the amount of wasted space
3. Don't make things so narrow cause strings to wrap unnecessarily
4. Try to crop to the border (including the border) for dialogs
5. Optimize the images with optipng (or pngquant, in some cases, although this second tool may drop quality in some cases a bit)

**Comment by garrett** [comment]
I changed them all in https://github.com/cockpit-project/cockpit-project.github.io/pull/265/commits/ce88cc15c607fd0137bd3aac67f2ba4b937f4305?short_path=201a304#diff-201a304f765aa8ed3f1529c089af8fa7

Perhaps I should vertically crop the service details image a bit more.

**Comment by garrett** [comment]
Force-pushed the previous commit with a better service details screenshot @ https://github.com/cockpit-project/cockpit-project.github.io/commit/0252421590588be0d8457ae9a8c5ff32fe57ec32?short_path=201a304#diff-201a304f765aa8ed3f1529c089af8fa7

**Comment by garrett** [comment]
TADA! We have the start of screenshot guidelines now: https://github.com/cockpit-project/cockpit-project.github.io/issues/183

These should be worked on a bit more and posted somewhere more official than in an issue. Meanwhile, they're in that issue if you'd like to refer to them. :wink:

**Comment by KKoukiou** [comment]
@garrett pls approve. your suggestions are in.

**Pull request merged**